### ISSUE TITLE
Issues with 0.107.x

### ISSUE THREAD

**Issue by SupremeSports** [thomasloven/lovelace-fold-entity-row#94]
Fold entity not working for input_select, section and probably others I might not be using. Sensors, lights, switches, etc works fine.

Updated fold-entity-row  to version 15 from HACS, updated card-tools

![8EB1D533-8615-429A-89F6-5070A7447E9C](https://user-images.githubusercontent.com/13887820/77130404-3ba60f80-6a2e-11ea-8069-4c240f72c3b5.jpeg)

PS: it works on iOS app but not in any windows browsers I tried (chrome, IE, Firfox, edge)

**Comment by thomasloven** [comment]
If a thing works in one browser and not in another, it's a caching issue in 99.96% of the cases.

Check your browser console. If it does not say "FOLD-ENTITY-ROW 1.0.1 IS INSTALLED ", clear your cache.

**Comment by SupremeSports** [comment]
Ok, so I cleared the cache, still not working, tried on cell phone browsers, not working. Tried another computer, not working. it is only working in the iOS app (not browsers).

For the browser console, sorry to ask, but I'm not sure where to check? I uninstalled my previous installation (manual) and reinstalled using HACS, I installed version 15... If I open fold-entity-row.js it has "version":"1.0.1" in it...
